README.md

```diff
@@ -66,7 +66,7 @@ See you soon with your first PR.
 Guidelines
 ----------
 
-We recommend you to introduce yourself on our [gitter channel](https://gitter.im/codezoned2017/Lobby). You can include the courses you have taken relevant to data structures and algorithms, some projects, prior experience, in your introduction. This will help us to allocate you issues of suitable difficulty.
+We recommend you to introduce yourself on our [gitter channel](https://gitter.im/codezoned2017/Lobby). You can include the courses you have taken relevant to data structures and algorithms, some projects, prior experience, in your introduction. This will help us to allocate your issues of suitable difficulty.
```

Comment:
```suggestion
We recommend you to introduce yourself on our [gitter channel](https://gitter.im/codezoned2017/Lobby). You can include the courses you have taken relevant to data structures and algorithms, some projects, prior experience, in your introduction. This will help us to allocate you issues of suitable difficulty.
```